README.md

```diff
@@ -86,6 +85,23 @@ To run the test suite with test weights (only works in branch `solutions`) while
 
 **NOTE**: all of these tasks tasks take the `currentProject` setting into account, see above.
 
+### Building docker container images for grading
+
+To build a docker container image to upload to Coursera to use for grading, be sure to have the following prerequisites installed:
+
+ - [`docker`](https://docker.com) - A popular container system for Linux.
+ - [`courseraprogramming`](https://github.com/coursera/courseraprogramming) - A toolkit to help instructional teams build grading containers
+
+To dockerize an sbt project, copy the sample docker file in `docker/Dockerfile.sample` to `Dockerfile` at the root of your project. Also copy `docker/grade.sh.template` to the root of your project. Then run: `docker build -t <your-project-name> .` to build and tag your container image (e.g. `docker build -t reactive .` to tag it as `reactive`).
+
+You can run `courseraprogramming inspect <your-project-name>` to launch an interactive terminal within the container image to poke at the file system, or run `courseraprogramming grade local <your-project-name> path/to/submission/folder` to run the grader on a sample submission. The submission folder should have a single jar inside named `submission.jar` containing the sources to grade. (You can generate this JAR by running `packageSubmission` in an student sbt project. Look for the resulting `jar` inside `target/scala-2.11` or display it by executing `show packageSubmission` in the `sbt` interactive console.)
```

Comment:
Nice - this looks very useful for debugging.
Again, in future steps, I'd introduce an SBT task for running the container image locally.